scss/_buttons.scss

```diff
@@ -154,7 +154,8 @@
     &,
     &:hover {
       color: #777;
-      background-color: #f5f5f5;
+      background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.